src/mono/mono/mini/mini-amd64.c

```diff
@@ -8046,6 +8063,38 @@ MONO_RESTORE_WARNING
 		}
 	}
 
+#if defined(AMD64_CALLEE_SAVED_XREGS)
+	if (method->save_lmf) {
+#if defined(MONO_HAVE_SIMD_REG)
+		int xreg_size = sizeof (MonoContextSimdReg);
+#else
+		int xreg_size = sizeof (double);
+#endif
+		save_area_offset = ALIGN_TO (save_area_offset, xreg_size);
+
+		for (guint16 i = 0; i < AMD64_XMM_NREG; ++i) {
+			if (AMD64_IS_CALLEE_SAVED_XREG (i)) {
+#if defined(MONO_HAVE_SIMD_REG)
+				amd64_movdqu_membase_reg (code, cfg->frame_reg, save_area_offset, i);
+#else
+				amd64_movsd_membase_reg (code, cfg->frame_reg, save_area_offset, i);
+#endif
+				if (cfg->arch.omit_fp) {
+					mono_emit_unwind_op_offset (cfg, code, AMD64_NREG + i, - (cfa_offset - save_area_offset));
+					/* These are handled automatically by the stack marking code */
+					mini_gc_set_slot_type_from_cfa (cfg, - (cfa_offset - save_area_offset), SLOT_NOREF);
+				} else {
+					mono_emit_unwind_op_offset (cfg, code, AMD64_NREG + i, - (-save_area_offset + (2 * 8)));
+					// FIXME: GC
```

Comment:
Re: `FIXME` - is there some follow up needed for this?

Reply:
That code is mainly a copy past, so kept the original comments as well, I guess that fixme have been in there for many many years, so no follow up have been planned, not even sure its relevant anymore, but if we drop it here, we should also drop it from the source where it got copied, https://github.com/dotnet/runtime/blob/c105e7da368ad23097e20e51b0ad054b771dab7c/src/mono/mono/mini/mini-amd64.c#L8041.

Reply:
I suspect it is some artifact from the times we had precise stack scanning.

Reply:
If stack scanning is always conservative (is that the case?) then there is nothing needed here for GC. 
Perhaps remove the `FIXME` from both places?